src/PanoImageRenderer/VideoLoader.js

```diff
@@ -0,0 +1,74 @@
+export default class VideoLoader {
+	constructor(video) {
+		this._isLoaded = false;
+		this._handlers = [];
```

Comment:
@happyhj I added event handler for managing content load event. Because if not managed, it fires event although it has been destroyed.